### ISSUE TITLE
Images alt attribute

### ISSUE THREAD

**Issue by ghost** [imathis/octopress#1097]
I think the builtin image elements should be updated so that they all have `alt` attributes. For example the g+ image that can be used to link to your profile. It is a W3C accessibility standard.

Reference: http://www.w3.org/QA/Tips/altAttribute
